### DISCUSSION TITLE
人生三階段

### DISCUSSION THREAD

**Discussion by Xunhaoz** [1101-GS4719-C/Python-Community-GS4719-C#1333] [Ideas]
我們從“Talk is cheap, show me the code”聊起。

這句話出自傳奇程序員Linus Torvalds， 他是Linux和Git（衍生出了著名的Github）之父，怎麼描述他的貢獻呢？整個互聯網世界，你作為一名小白用戶，在前台看到的都是蘋果微軟的產品，在你看不到地方，整個互聯網的“大後台”，幾乎全是由Linux驅動的。他還是開源運動的領軍人物，從一開始就拒絕將這些產品商業化，非常“Negative”。在計算機領域，是身處鄙視鏈頂端的大神，地位在蓋茨和喬布斯之上。

推薦大家去看看Linus的自傳，如果沒耐心，只看第一章也行。在第一章中，他與傳記作者閒聊，說自己一直在思考一個問題——生活的意義是什麼？他略帶調侃地說出了答案：

- 我對生活的意義有些看法。咱們可以在第一章跟讀者說一下生活的意義來釣他們上鉤，等他們上鉤，花錢買書之後，我們再隨便扯點什麼把剩下的章節糊弄過去……有三件事是對生活有意義的，它們是生活中所有事情的動機——包括你做的所有事和任何一個生命體會做的事：第一是生存，第二是社會秩序，第三娛樂。生活中所有的事都遵循著這個順序，娛樂之後就再無其他。所以某種意義上來說，生活的意義就是要你達到第三個階段。一旦達到了第三個階段，這輩子你就算成功了。但是你得先超越前兩個階段。 （Linus《只是為了好玩》）

他舉了個例子，比如性，性一開始是為了生存繁衍，然後被社會化，變得與法定年齡、道德倫理和婚姻制度有關，最後性變成了一種娛樂和遊戲。

有意思的是，Linus漫不經心道出的三階段，恰好呼應了阿倫特在《人的境況》中概括人的三類活動：勞動-工作-行動。

生產與勞動、社會秩序與工作的對應無須解釋，阿倫特所說的行動與遊戲有很大交集，它與創造和自由有關，不過分學究的話，可以認為他們倆說的是一回事。

阿倫特說得很深刻：

- 去行動，在最一般的意義上，意味著去創新、去開始，發動某件事。而人就他的誕生而言是initium——新來者和開創者，人能開端啟新。這個開端不同於世界的開端，開端不是某物的，而是某人的，人自身就是一個開創者。由於人的被造，開端原則才進入了本然世界，當然這就等於說，自由原則在人被造的那一刻才出現……人能夠行動的事實，意味著總是可以從他身上期待未曾預料的事情，他能夠完成不可能的任務。而這一點又之所以是可能的，僅僅因為每個人都是獨特的，每個人的誕生都為世界帶來獨一無二的新東西。就這個人是獨一無二的而言，真的可以說在他之前，無人在此。 （漢娜·阿倫特《人的境況》）

人生來就是為了好玩，成長磨礪是為了復歸天真。受制於外部條件，我們求生存，為求生存，我們結黨成群，有群必有群規，隨後社群規模擴大，變成社會，群規成了秩序和律法，最終系統世界殖民生活世界，這才有了我們的當前的窘境，一邊吐槽自己乾著“毫無意義的狗屁工作”，但又擔心這個狗屁工作被機器人搶走，在一個更大的尺度上，工作不是天經地義的，它本身是需要被跨越的階段。

如今，我們站在電時代與智時代的檻界上。每一次技術革命都將消滅那些原本無意義、卻受制於時代局限而不得不做的工作。曾經是農民、後來是工人，今天就是白領。白領們的認知勞動在語言層次展開，語言具有無限延展性，不存在生理與物理的極限。因此，電時代白領們的認知勞動天然帶有“卷”的屬性，而未來的認知勞動不是卷不捲的問題，而是直接失去價值了。在這個意義上，AI不會加劇內捲，它是內捲終結者。

**Comment by boson13579** [comment]
我要說這是一個非常有趣和富有啟發性的文章，它從不同的角度探討了生活的意義、工作和創新的關系，以及技術革命對人類的影響。我很樂意分享我的看法，並談談我會如何應對這些問題。
首先，關於生活的意義，我同意Linus Torvalds的看法

**Comment by boson13579** [comment]
他認為生活的意義在於追求娛樂和享受。生存和社會秩序是實現娛樂的基礎，但娛樂才是我們最終的追求。對我而言，生活的意義在於尋找快樂和滿足感，追求個人的興趣和激情，建立有意義的人際關系，並為自己和他人創造美好的回憶和體驗。

**Comment by boson13579** [comment]
關於工作和創新，我認為阿倫特和Linus Torvalds所提到的概念是相關的。工作和勤勞是我們在生活中不可或缺的一部分，它們確保我們的生存和社會秩序。然而，真正有意義和充實的生活還需要行動和創新。通過行動和創新，我們能夠突破傳統界限，開創新的可能性，創造獨特的價值，並對自己和周圍的世界做出積極的影響。

**Comment by boson13579** [comment]
在面對電子時代和智能時代的挑戰時，我認為我們應該持開放的態度並適應變化。技術革命確實會改變工作和社會結構，但它也為我們創造了新的機會和可能性。我會繼續探索和理解這些變化，研究其對個人和社會的影響，並尋找創新的方式來應對這些變化。

**Comment by boson13579** [comment]
具體而言，我會：
持續學習和適應：了解新技術和趨勢，學習相關知識和技能，以保持與時俱進。
探索個人興趣：發展自己的興趣愛好，並將其與工作相結合，尋找內在的動力和滿足感。
培養創新思維：鼓勵自己思考問題的不同角度，並嘗試新的方法和解決方案，以促進個人和社會的進步。

**Comment by boson13579** [comment]
建立有意義的人際關系：與他人合作、分享和交流，尋找共同的目標和合作機會，以實現更大的影響力和成就感。
總之，生活的意義是一個深刻而覆雜的問題，每個人都可能有不同的回答。我的目標是探索這些問題，並通過積極的行動和創新來實現個人和社會的進步。
